ecc/bls12-381/hash_vectors_test.go

```diff
@@ -9,6 +9,7 @@ import (
 func TestG1IsogenyVectors(t *testing.T) {
 	t.Parallel()
 
+	// TODO @gbotrel fix me test vectors shouldn't set words directly
```

Comment:
Right, if these come from the standard docs they should be available in decimal or hex form.